README.md

```diff
@@ -24,7 +24,7 @@ git clone https://github.com/tud-cor/fs_mod_ros_windows.git
 #### Creating the symbolic link
 
 In order to exchange data with the Python side of the mod, the script uses a symbolic link to a *named pipe*.
-This named pipe needs to be created.
+This named pipe needs to be created *only once*. Do not create it every time the FarmSim is rebooted.
```

Comment:
The *pipe*, or the *symlink*?

The pipe is created by the Python script, correct? The symlink is what `mklink` is used for.

Reply:
yes, it's *symlink*. Fixed it